bin/express-cli.js

```diff
@@ -147,12 +148,32 @@ function createApplication (name, dir) {
     dependencies: {
       'debug': '~2.6.9',
       'express': '~4.16.1'
+    },
+    devDependencies: {
     }
   }
 
+  if (program.typescript) {
+    pkg.scripts['build'] = 'tsc'
+
+    pkg.devDependencies['typescript'] = '~3.7.5'
+    pkg.devDependencies['@types/node'] = '~13.7.0'
+    pkg.devDependencies['@types/debug'] = '~4.1.5'
+    pkg.devDependencies['@types/express'] = '~4.17.2'
```

Comment:
Should the types package version pach the express package version?

Reply:
Agree, better align with express v4.16.1.
Per check, @types/express also support typing on v4.16.